sigal/plugins/titleregexp.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022 - C.Sehrt
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to
+# deal in the Software without restriction, including without limitation the
+# rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+# sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+# IN THE SOFTWARE.
+
+""" This plugin modifies titles of galleries by using regular-expressions and
+simple character replacements.
+
+Settings:
+
+- ``titleregexp`` with the following keys:
+    - ``regexp``, which is an array of dicts with 'search', 'replace' and 
+        'count' keys
+    - ``substitute``, which is an array of 2-element-arrays, of which the
+        second element denotes the replacement of occurences of the first 
+        element
```

Comment:
Could you explain a bit more what the regex is doing ? (I see it's using `re.subn` but it's not clear from the docs)